cranelift/codegen/src/isa/s390x/inst/mod.rs

```diff
@@ -3188,8 +3188,8 @@ impl Inst {
             &Inst::Debugtrap => "debugtrap".to_string(),
             &Inst::Trap { .. } => "trap".to_string(),
             &Inst::TrapIf { cond, .. } => {
-                let cond = cond.invert().pretty_print_default();
-                format!("j{} 6 ; trap", cond)
+                let cond = cond.pretty_print_default();
+                format!("jg{} .+2", cond)
```

Comment:
Like the other backends (which Jamey suggested and I think is a good idea), could this include the trap code in the printing? The other backends do something like `# trap={code}` at the end IIRC

Reply:
Indeed, good idea.  I've also added this for the unconditional traps now.